scripts/debug_navigation_native.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# This script replaces SPM checked out MapboxNavigationNative framework for simulator with a debug version.
+
+if [[ -z $1 || -z $2 ]]; then 
+    echo "Usage $0 <path to MapboxNavigationNative git repository> <simulator xcodebuild destination>"
+    exit 1
+fi
+
+set -x
+set -e
+
+# Install a tool which parses JSONs
+brew list jq || brew install jq
+
+NATIVE_PATH=$1
+DESTINATION=$2
```

Comment:
Can we use "any simulator" here?

Reply:
"generic/platform=iOS Simulator" works. Thank you